suite-native/module-onboarding/src/screens/FeatureReceiveScreen.tsx

```diff
@@ -7,82 +9,71 @@ import {
     StackNavigationProps,
 } from '@suite-native/navigation';
 import { Box } from '@suite-native/atoms';
-import { useTranslate } from '@suite-native/intl';
+import { TxKeyPath, useTranslate } from '@suite-native/intl';
 
 import { OnboardingFooter } from '../components/OnboardingFooter';
 import { OnboardingScreen } from '../components/OnboardingScreen';
 import { CoinsSvg } from '../components/CoinsSvg';
 
-const FeatureReceiveScreenWithDevice = () => {
-    const { translate } = useTranslate();
+type NavigationProps = StackNavigationProps<
+    OnboardingStackParamList,
+    OnboardingStackRoutes.AboutReceiveCoinsFeature
+>;
+
+type ScreenContent = {
+    title: TxKeyPath;
+    subtitle: TxKeyPath;
+    redirectTarget: OnboardingStackRoutes;
+};
+const receiveScreenContentMap = {
+    usb: {
+        title: 'moduleOnboarding.featureReceiveScreen.usb.title',
+        subtitle: 'moduleOnboarding.featureReceiveScreen.usb.subtitle',
+        redirectTarget: OnboardingStackRoutes.TrackBalances,
+    },
+    noUsb: {
+        title: 'moduleOnboarding.featureReceiveScreen.noUsb.title',
+        subtitle: 'moduleOnboarding.featureReceiveScreen.noUsb.subtitle',
+        redirectTarget: OnboardingStackRoutes.AnalyticsConsent,
+    },
+} as const satisfies Record<'usb' | 'noUsb', ScreenContent>;
```

Comment:
What about change the `usb | noUsb` to something like `device | portfolioTracker`? I don't like usb, because in the feature it won't be the only way how to connect device. Just a tiny nit, won't block the PR because of this.

Reply:
[e774f1c](https://github.com/trezor/trezor-suite/pull/10582/commits/e774f1cf85d1f956857b77e409db6ef14cfc8c5a)